### PR TITLE
RFC: Update Travis configuration

### PR DESCRIPTION
**Pull request by darwindarak**
`Plots.jl` tests are actually running now, instead of failing during installation.  The tests seem to be failing because the images produced are actually slightly different from the reference ones.  For example:

The reference image
ref1

compared to the generated image
ref1

Maybe due to the changes in Matplotlib 2.0?

### CODE REVIEW

**Comment by mkborregaard** [comment]
Yes, see https://github.com/JuliaPlots/Plots.jl/issues/670 . That issue is not related to the PR, though, right?

**Comment by pkofod** [comment]
This includes v0.6 which is fine, but what is actually solved here? Sorry for being dense.

**Comment by darwindarak** [comment]
@mkborregaard You're right, that is a separate issue.  This PR makes it so that the tests at least get to running some Julia code.  It was stuck installing `wkhtmltox`.

@pkofod Maybe nothing, since even tests on 0.5 have been ignored for the last 4 months.  But 0.6 is around the corner and there seems to be a couple 0.6 specific bugs that make Plots unusable on almost any backend.  Whether these tests pass or fail, they provide useful information for debugging.  For example, the test [results](https://travis-ci.org/JuliaPlots/Plots.jl/jobs/230019825) show the PyPlot backend working just fine.  It turns out that setting the Matplotlib backend to `Agg` bypasses the [world age problem](https://github.com/JuliaPlots/Plots.jl/issues/734).

File: test/install_wkhtmltoimage.sh [hunk 1]
```diff
@@ -7,7 +7,8 @@ sudo apt-get -qq update
 
 sudo apt-get install -y xfonts-75dpi
 wget http://download.gna.org/wkhtmltopdf/0.12/0.12.2/wkhtmltox-0.12.2_linux-trusty-amd64.deb
-sudo dpkg -i wkhtmltox-0.12.2_linux-trusty-amd64.deb
+sudo dpkg -i wkhtmltox-0.12.2_linux-trusty-amd64.deb || true
```
Comment (tkelman) [inline test/install_wkhtmltoimage.sh]: why was this failing, wouldn't it be better to address the issue instead of letting it fail and ignoring it?

Comment (darwindarak) [inline test/install_wkhtmltoimage.sh]: It was missing a dependency, which the next line resolves automatically.  I figured that if that doesn't work, then the tests will still fail when it tries to run `wkhtmltoimage`.

Comment (tkelman) [inline test/install_wkhtmltoimage.sh]: ah. debs are screwy. is there a specific flag to dpkg that could tell it to only ignore dependency issues?

Comment (darwindarak) [inline test/install_wkhtmltoimage.sh]: You're right, there's a `--depends` flag to turn all dependency issues into warnings.

**Comment by mkborregaard** [comment]
@daschw What do you say to this PR? Is the `force-depends` an alternative way of solving the issues we've had with wkhtmltoimage? I'm for adding a 0.6 test now. I don't think we should remove the checkout of PlotUtils, but we can install StatPlots normally.

**Comment by daschw** [comment]
>  Is the force-depends an alternative way of solving the issues we've had with wkhtmltoimage?

We had 2 issues with wkhtmltoimage. 1. The server http://download.gna.org/... was/is not available. So I changed the link. 2. There was a missing dependency for wkhtmltox (xfonts-base). I added this dependency [here](https://github.com/JuliaPlots/Plots.jl/blob/master/test/install_wkhtmltoimage.sh#L8), and @darwindarak turned the dependency error to a warning with `--force-depends` and then installed all dependencies with `sudo apt-get -f install`.

>  I don't think we should remove the checkout of PlotUtils, but we can install StatPlots normally.

I agree. I don't know what the reason for removing `Pkg.checkout("PlotUtils")` was, though.

**Comment by mkborregaard** [comment]
All in all I think the conclusion is that this PR can be closed. Thanks for the PR, though, @darwindarak nothing wrong with it we just chose a different way around the issues.

**Pull request closed without merging**